packages/ui/src/working-groups/types/WorkingGroup.ts

```diff
@@ -40,3 +40,12 @@ export const asWorkingGroup = (group: WorkingGroupFieldsFragment): WorkingGroup
     budget: new BN(group.budget),
   }
 }
+
+export type GroupName = 'forum' | 'storage' | 'content' | 'membership'
```

Comment:
We might introduce configuration for this so it might come from a JSON in the future :thinking: 

Right now I think that we can have 

```typescript
const KnowWorkingGroups = ['Membership', 'Storage', 'Other'] as const

type GroupName = typeof KnowWorkingGroups[number]

const isGroupname = ( name: string ): name is GroupName => {
  return KnowWorkingGroups.includes(name as any)
}
```